### DISCUSSION TITLE
Behaviour of landscape with given page height/width

### DISCUSSION THREAD

**Discussion by brdvd** [rism-digital/verovio#3306] [General]
Suppose in Verovio I want to have a page in landscape that is extra wide. The straightforward approach would be `--landscape  --page-width 5000`. 

However, to get the expected result, one needs to use `--landscape  --page-height 5000`, since Verovio takes the page height/width options first and applies a rotation afterwards depending on landscape.

What would be the desired behaviour of the `--landscape` option with given `--page-width/--page-height` from a user perspective?

**Comment by lpugin** [comment]
I agree this is confusing. The --landscape option actually only swaps height and width. It is a very old option. Maybe we should just deprecate/remove it?

**Reply by brdvd** [reply]
Actually I use it a lot (I might be the only person). Maybe it suffices to clarify the behaviour in the description of the option, i.e. that it swaps the page height and width values.
